workspace_api/views.py

```diff
@@ -98,7 +99,20 @@ async def create_workspace(
         )
     )
 
-    create_bucket(workspace_name=workspace_name)
+    # create_bucket(workspace_name=workspace_name)
+    # TODO: specify post body
+    response = requests.post(bucket_endpoint_url, data={})
```

Comment:
Missing request body, see values here: https://github.com/EOEPCA/rm-workspace-api/wiki/Workspace-representation-in-Kubernetes#k8s-bucket-claim